edgelet/iotedge/src/system.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use std::ffi::OsStr;
+
+use crate::error::{Error, ErrorKind};
+use aziotctl_common::{
+    get_status, get_system_logs as logs, restart, set_log_level as log_level, ServiceDefinition,
+    SERVICE_DEFINITIONS as IS_SERVICES,
+};
+
+lazy_static! {
+    static ref IOTEDGED: ServiceDefinition = {
+        let sockets: &'static [&'static str] = option_env!("IOTEDGE_HOST").map_or(&[], |_host| {
+            &["aziot-edged.mgmt.socket", "aziot-edged.workload.socket"]
+        });
+
+        ServiceDefinition {
+            service: "aziot-iotedgd.service",
+            sockets,
+        }
+    };
+    static ref SERVICE_DEFINITIONS: Vec<&'static ServiceDefinition> = {
+        let iotedged: &ServiceDefinition = &IOTEDGED;
```

Comment:
since IOTEDGED is only used from this `static ref`, you aught to just inline the definition.

Reply:
What do you mean by inline it? SERVICE_DEFINITIONS is a vec of refs, so the Service_Definition (IOTEDGED in this case) needs to be static in order to live long enough.

Reply:
ahhhh, true. okay, carry on.